flocker/acceptance/test_api.py

```diff
@@ -0,0 +1,96 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+Tests for the control service REST API.
+"""
+
```

Comment:
Or just check every string and make sure it's either b"" or u"" and not just "".